Common Programming Concepts/Data Types/Understanding Types/task.md

```diff
@@ -0,0 +1,10 @@
+## Mastering the IDE: Understanding Types
+
+Let's examine the code from the [Slices](course://Common Programming Concepts/Data Types/Slices) task more closely.
```

Comment:
The Slices task [was moved](https://github.com/jetbrains-academy/rustlings-course/commit/e64b1b16b2e19e2ca865550c911a05d8fd8ac850) further in the curriculum, into the Understanding Ownership - Move Semantics lesson. I suggest removing the link and not referring to a parent at this point.